README.md

```diff
@@ -0,0 +1,141 @@
+# Rootless Pixel Launcher
+
+![Rootless Pixel Launcher in action](https://i.imgur.com/wTCa3uW.gif)
+
+The official Pixel Launcher has a lot of exclusive features like the Google Now panel and G Search Pill that only work while it is running as a system app. When regular users install the APK file they get a gimped version of the app that refuses to give them these features. However, after seeing the ParanoidAndroid commits to get the Google Now panel working I realized these were not limitations of being installed as a normal app, instead Google intentionally made it work this way, most probably to keep the features exclusive to the Pixel phones as a selling point like HDR+. The older Google Now Launcher could make use of the Google Now panel too when you installed it from the Play Store.
+
+--------
+
+**Downloads**: https://github.com/amirzaidi/Launcher3/releases
+
+**Pictures**: https://goo.gl/photos/C6kWrgMV3jMfL2Ld9
+
+**Sources**: https://github.com/amirzaidi/Launcher3/commits/o
+
+*Report bugs with a logcat, if you don't know how to take one please use Google. I do not ask for donations and I did not add ads in the launcher. If you want to support me, wish me a happy birthday on September the 6th.* 
+
+--------
+
+## Version 2.1 is out!
+
+- Backports: transparent hotseat, Search Apps layout, transparent QsbConnector
+
+- Automatically hide the Google Now feed page and Google Pill when the Google App is disabled
+
+--------
+
+**Pixel Launcher features added to Launcher3**
+
+- QSB with transparent rectangle, Google Pill and Date/Weather
+
+- Different colours of notification dots
+
+- Filter Google Wallpapers and Voice Search from the apps list
+
+- Use Google Wallpapers to select a wallpaper when available
+
+- Device profiles, margins, icon counts
+
+- Disabled partner customization like Motorola's 4 columns
+
+- Google Now feed on the left of the normal workspace
+
+- Google Calendar app icon with the date
+
+--------
+
+**My additions on top of the basic Pixel Launcher ones**
+
+- On popular request for the Nougat version: Pull down for notifications
+
+- Oreo theme backported to older OS versions with Pixel Blue accent colour
+
+- Notification dots backported to Marshmallow
+
+- Automatically prompts for notification access so you won't have to dig through settings menus
+
+- Fix Android 8 checks so Nexus devices can use all new features
+
+- Show icon shapes on Android 8 without developer settings enabled
+
+- Pressing the date widget opens the default calendar app
+
+- Keyboard properly closes when returning to the home screen from any app
+
+- Symmetrical hotseat
+
+- Filter Google Now Launcher from the apps list
+
+- Pinch to overview
+
+- Samsung Secure Folder compatibility
+
+- Backport of circle icons
+
+From my previous post:
```

Comment:
Probably confusing without reddit context

Reply:
Agree. At the time I just copied and pasted from Reddit, so it still needs some work.